types/jasmine-given/index.d.ts

```diff
@@ -1,7 +1,8 @@
 // Type definitions for jasmine-given 2.6
 // Project: https://github.com/searls/jasmine-given
-// Definitions by: Shai Reznik <https://github.com/shairez>
+// Definitions by: Shai Reznik <https://github.com/shairez>, Dmitry Efimenko <https://github.com/DmitryEfimenko>
```

Comment:
Let's divide into separate lines for more convenient reading?

```ts
// Definitions by: Shai Reznik <https://github.com/shairez>
//                 Dmitry Efimenko <https://github.com/DmitryEfimenko>
```

Reply:
Updated